plugins/modules/site_workflow_manager.py

```diff
@@ -1625,6 +1624,45 @@ def change_payload_data(self, config):
             self.msg = "Unable to process the payload data : {}".format(str(e))
             self.set_operation_result("failed", False, self.msg, "ERROR").check_return_status()
 
+    def is_site_exist(self, site_name):
+        """
+        Checks if a site exists in Cisco Catalyst Center by retrieving site information based on the provided site name.
+
+        Args:
+            site_name (str): The name or hierarchy of the site to be retrieved.
+
+        Returns:
+            tuple (bool, str or None): A tuple containing:
```

Comment:
Docstring - returns a tuple (bool, str or None).
However, in the implementation, it only returns site_exists, which is a single value (either None or True)